### ISSUE TITLE
Menu in presentation view

### ISSUE THREAD

**Issue by bwulff** [slidewiki/slidewiki-platform#1192]
- the menu in the presentation view (reveal.js) ist in parts not usable at all with a screen reader
- the menu seems to have little use at all since the user can go back to the slidewiki view in order to browse the deck structure

--> proposal: remove the menu from the presentation view

**Comment by aoelen** [comment]
Can you tell me which parts of the menu are not accessible? I think the menu can be useful to browse through large decks of slides

**Comment by bwulff** [comment]
This is low prio as it is actually not our code base. Let's discuss on the meeting in Halle.

**Comment by aoelen** [comment]
I've added an aria-hidden attribute to the menu (now the menu is not read by a screen reader). See repo: https://github.com/slidewiki/reveal.js-menu. The menu itself is still not very accessible.

It is possible to remove the menu all together. Let me know what you think.

**Comment by bwulff** [comment]
Having tried it I don't see a lot of value in keeping the menu. removing it
might even be beneficial since it simplifies things for everyone ... let's
try it then, @aoelen can you make a seperate branch with the menu removed,
please?

**Comment by aoelen** [comment]
Ok, I will do this. I am all in for simplifying things!
